docs(build-performance): update ts-loader section (#3851)
webpack/webpack.js.org: src/content/guides/build-performance.md

### TypeScript Loader

> Faster builds using ts-loader can be acheived by using the `transpileOnly` loader option. On its own, this option turns off typechecking. To gain typechecking again, use the [`ForkTsCheckerWebpackPlugin`](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin). This speeds up TypeScript type checking and ESLint linting by moving each to a separate process.

Reviewer: ```suggestion
Faster builds using `ts-loader` can be achieved by using the `transpileOnly` loader option. On its own, this option turns off type checking. To gain type checking again, use the [`ForkTsCheckerWebpackPlugin`](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin). This speeds up TypeScript type checking and ESLint linting by moving each to a separate process.
```

> Faster builds using `ts-loader` can be achieved by using the `transpileOnly` loader option. On its own, this option turns off type checking. To gain type checking again, use the [`ForkTsCheckerWebpackPlugin`](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin). This speeds up TypeScript type checking and ESLint linting by moving each to a separate process.

Reviewer: ```suggestion
To improve the build time when using `ts-loader`, use the `transpileOnly` loader option. On its own, this option turns off type checking. To gain type checking again, use the [`ForkTsCheckerWebpackPlugin`](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin). This speeds up TypeScript type checking and ESLint linting by moving each to a separate process.
```

> There is a [full example](https://github.com/TypeStrong/ts-loader/tree/master/examples/fork-ts-checker-webpack-plugin) on the ts-loader github repository.

Reviewer: ```suggestion
```

    {
      loader: 'ts-loader',
      options: {
        transpileOnly: true
      },
    },
  ],
};
```

> ---

Reviewer: ```suggestion
T> There is a [full example](https://github.com/TypeStrong/ts-loader/tree/master/examples/fork-ts-checker-webpack-plugin) on the `ts-loader` GitHub repository.

---
```
